src/TraceEvent/Computers/StartStopActivityComputer.cs

```diff
@@ -1006,6 +1009,54 @@ bool TryProcessDiagnosticSourceStartEvents(TraceEvent data)
             return false;
         }
 
+        void FixAndProcessAppInsightsEvents(TraceEvent data)
+        {
+            Debug.Assert(data.ProviderGuid == MicrosoftApplicationInsighsDataProvider);
+            Debug.Assert(data.Opcode == TraceEventOpcode.Start || data.Opcode == TraceEventOpcode.Stop);
+
+            if (data.TaskName.Equals("Request", StringComparison.OrdinalIgnoreCase))
```

Comment:
❔ Does this really come in different cases? If not, can we use the correct case here and switch to the faster `StringComparison.Ordinal`?